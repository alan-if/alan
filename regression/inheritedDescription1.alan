Every little_thing Isa object
  Is little.
  Description
	"There is a "
	If This Is little Then "little" End If.
	"thing here."
End Every little_thing.

The first_thing Isa little_thing At l
End The first_thing.

The second_thing Isa little_thing At l
  Is Not little.
End The second_thing.

The l Isa location
End The l.

Start At l.


