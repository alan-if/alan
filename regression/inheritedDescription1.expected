########## inheritedDescription1 ##########

        No warnings or errors detected.



L
There is a little thing here. There is a thing here.

> 

Do you want to UNDO, RESTART, RESTORE or QUIT ? 
